config/provider.go

```diff
@@ -152,144 +25,31 @@ var providerMetadata string
 // GetProvider returns provider configuration
 func GetProvider() *ujconfig.Provider {
 	pc := ujconfig.NewProvider([]byte(providerSchema), resourcePrefix, modulePath, []byte(providerMetadata),
-		ujconfig.WithIncludeList(ExternalNameConfigured()),
```

Comment:
Should we remove `ExternalNameConfigured()` func in `externalname.go` then?

Reply:
thanks done